docker-compose.yml

```diff
@@ -34,7 +34,9 @@ services:
     ports:
       - "8072:8000"
     volumes:
-      - .:/edx/app/xblock
+      # Override installed xblock with the xblock source directory to provide an
+      # editable environment
+      - ./configurable_lti_consumer:/usr/local/lib/python2.7/dist-packages/configurable_lti_consumer
```

Comment:
@jmaupetit you just forgot to do the same on the LMS. Otherwise... it works :tada: !